### ISSUE TITLE
App is not showing arpoints

### ISSUE THREAD

**Issue by mescalantea** [dat-ng/ar-location-based-android#8]
I am geting right gps location but no arpoints are rendered on camera rotation. Maybe the sensor implementation?? I have a Moto G4 play runing Android Marshmallow

**Comment by dat-ng** [comment]
I think your phone doesn't have compass sensor. Please check

2017-07-24 22:44 GMT+07:00 mescalantea <notifications@github.com>:

**Comment by mescalantea** [comment]
You are right. Do you know another way to achieve this using others sensors??

**Comment by dfsanabria** [comment]
What is the difference between the compass and the gyroscope in this project?

**Comment by SamplesCZ** [comment]
@mescalantea hey did you find any solution..
